contracts/Aavegotchi/ForgeDiamond/facets/ForgeViewsFacet.sol

```diff
@@ -0,0 +1,94 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.1;
+
+import "../libraries/LibAppStorage.sol";
+
+contract ForgeViewsFacet is Modifiers {
```

Comment:
Contains all the new forge view functions that will be needed to fetch important forge-specific data for migration